tests/polkadotjs_test/test_copy.js

```diff
@@ -0,0 +1,112 @@
+// Import
+const { ApiPromise, WsProvider } = require('@polkadot/api');
+const {Keyring } = require('@polkadot/keyring');
+
+async function main() {
+    // Construct
+
+    const wsProvider = new WsProvider('ws://127.0.0.1:8546');
+    // const wsProvider = new WsProvider('ws://127.0.0.1:9944');
```

Comment:
I've removed test_copy.js entirely since this was a work in progress file to experimenting with javascript tests.  I've re-created relevant tests in tests/polkadotjs_test/test/test-polkadot.js where they are nun in a mocha test framework, which is more CI friendly.